### PR TITLE
PEP landing page

### PR DESCRIPTION
**Pull request by jbothma**
Still needs

- call to action
- expansion on data quality approach
- actual implementation instead of hardcoding copy pasta html

### CODE REVIEW

**Comment by pudo** [comment]
One cool things we could do here is also to run a facet query for countries on the search, eg. '/search/peps?facet=countries&schema=Person', then use that to show a country list with rough totals.

File: app/pep/page.tsx [hunk 1]
```diff
@@ -0,0 +1,208 @@
+import Link from 'next/link'
+import styles from '../../styles/Home.module.scss'
```
Comment (jbothma) [inline app/pep/page.tsx:2]: generalise the classes used or something

File: app/pep/page.tsx [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+          </Col>
+        </Row>
+      </Container>
+
+      <div className={styles.commercialBanner} id="get-the-data">
+        <Container>
+          <h2>Use OpenSanctions to manage business risk</h2>
+          <Row>
+            <Col md={8}>
+              <p>The PEP dataset is available for commercial licensing as part of the OpenSanctions Default dataset bulk data license and SaaS API.</p>
```
[the remark this answers is not in the document]
Comment (jbothma) [reply app/pep/page.tsx:190]: Screenshot_2023-06-15_16-43-58

File: app/pep/page.tsx [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+import { Col, Row, Container, Badge, Button, ButtonGroup, Table } from '../../components/wrapped';
+import LayoutFrame from '../../components/layout/LayoutFrame';
+import { SourcesTableProps } from '../../components/Dataset';
+import { FormattedDate, Numeric, UnofficialBadge, Spacer } from '../../components/util';
+import { getDatasets, getDatasetByName } from '../../lib/data';
+import { isCollection, isSource } from '../../lib/types'
+import DatasetCountryListing from '../../components/DatasetCountryListing';
+
+
+function SourcesTable({ sources }: SourcesTableProps) {
```
Comment (jbothma) [inline app/pep/page.tsx:12]: if we're building our own sources table here, don't reuse SourcesTableProps for the original SourcesTable

File: app/pep/page.tsx [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+  const sourcesSorted = sources.sort((a, b) => b.target_count - a.target_count)
+  return (
+    <div className="table-responsive">
```
Comment (jbothma) [inline app/pep/page.tsx:15]: double-check if there are unintended consequences of using table-responses

File: app/pep/page.tsx [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+      <Table size="sm">
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Last changed</th>
+            <th className="d-none  d-sm-table-cell">Publisher locality</th>
```
Comment (jbothma) [inline app/pep/page.tsx:21]: do we generally use bootstrap classes like this?

File: app/pep/page.tsx [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+              Find Politically Exposed Persons (PEPs) worldwide
+            </h1>
+            <p className={styles.subClaim}>
+              We consolidate information about public office holders from most countries in the world into an easy-to-use dataset.
+            </p>
+            <p><a href="#get-the-data"><Button variant="light">Get the data</Button></a></p>
+
+          </Row>
+        </Container>
+      </div>
```
Comment (jbothma) [inline app/pep/page.tsx:81]: the vertical spacing here doesn't feel right

File: app/pep/page.tsx [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+      <Container>
+        <Row>
+          <Col md={4} className={styles.explainer}>
+            <h2>Data Source Compiling</h2>
+            The OpenSanctions PEPs dataset packages publicly available PEP data
+            from various sources in a consistent structured format. <a href="#sources">Read
```
Comment (jbothma) [inline app/pep/page.tsx:87]: check if I should be using <Link> for in-page anchors too

File: app/pep/page.tsx [hunk 8]
```diff
@@ -0,0 +1,208 @@
...
+              more about our data PEP sources</a> and how
+            they are maintained.
+          </Col>
+          <Col md={4} className={styles.explainer}>
+            <h2>Cross-referenced and merged</h2>
```
Comment (jbothma) [inline app/pep/page.tsx:92]:
```suggestion
            <h2>Cross-referencing and deduplication</h2>
```

File: app/pep/page.tsx [hunk 9]
```diff
@@ -0,0 +1,208 @@
...
+            Over time our PEP data is <Link href={"/docs/enrichment/"}>enriched</Link> from various sources. Entities are
+            cross-referenced and where possible, multiple references to the same
+            real world entity are <Link href={"/docs/identifiers/"}>merged to a single identifier</Link>.
```
Comment (jbothma) [inline app/pep/page.tsx:95]: double-check "merged to a single identifier" wording on destination

File: app/pep/page.tsx [hunk 10]
```diff
@@ -0,0 +1,208 @@
...
+          </Col>
+          <Col md={4} className={styles.explainer}>
+            <h2>PEPs data as a commodity</h2>
+            Our ultimate objective is not to build another proprietary PEPs database,
+            but to expand the coverage of political persons in Wikidata such that it
+            becomes a de-facto global commodity.
+          </Col>
+        </Row>
+        <Row style={{ marginTop: "2em" }}>
```
Comment (jbothma) [inline app/pep/page.tsx:104]: is there a utility class or other more maintainable way than style attribute to get this spacing right?

File: app/pep/page.tsx [hunk 11]
```diff
@@ -0,0 +1,208 @@
...
+            </Row>
+            <Row>
+              <section>
+                <h2 id="sources">Where is our PEP data sourced from?</h2>
+
+                <p>OpenSanctions does not monitor political events and capture changes
+                  in roles manually based on election results or appointment press
+                  releases. Instead we automatically monitor and import updates from
+                  a set of public datasets of Politically Exposed Persons with various
+                  degrees of geographic influence coverage.</p>
```
Comment (jbothma) [inline app/pep/page.tsx:128]:
```suggestion
                  degrees of coverage of influence in various countries.</p>
```

File: components/DatasetCountryListing.tsx [hunk 12]
```diff
@@ -10,39 +10,54 @@ import { HelpLink, Numeric, Plural, Spacer } from './util';
...
+                  {c.label}
+                </a>
+              </td>
+              <td className="numeric"><Numeric value={c.count} /></td>
+            </tr>
+          )}
+        </tbody>
+      </Table>
+      {coverageExpanded && limit !== null && (
+        <a onClick={(e) => { e.preventDefault(); setLimit(null) }} href='#'>Show all...</a>
```
Comment (jbothma) [inline components/DatasetCountryListing.tsx:59]: This needs some spacing or something - the positioning feels off

File: app/pep/page.tsx (hunk 5 above)
Comment (jbothma) [reply app/pep/page.tsx:21]: similar display classes used on homepage

File: app/pep/page.tsx (hunk 7 above)
Comment (jbothma) [reply app/pep/page.tsx]: nope - <Link href="#id"> havigates to the homepage?!

File: app/pep/page.tsx (hunk 9 above)
Comment (jbothma) [reply app/pep/page.tsx:98]: I think a "single identifier" carries more meaning in this context than "single entity" - it's a bit more concrete - and it's not really technically incorrect. Even though the headline uses "merged entity" on the destination.

File: components/DatasetCountryListing.tsx (hunk 12 above)
Comment (jbothma) [reply components/DatasetCountryListing.tsx:59]: fixed by allowing the table to have its default margin when it's not nested

**Pull request merged**